### PR TITLE
Added back the "datasource" attribute to the examples for V4

### PR DESCRIPTION
**Pull request by alonsogarciapablo**
I basically removed a bunch of code from vis/vis.load that was there to make createVis work when no "datasource" attribute was present in the viz.json and added that attribute to the viz.json in the examples in the new v4 folder.

cc: @viddo

### CODE REVIEW

**Comment by alonsogarciapablo** [comment]
ping @viddo!

File: src/vis/vis.js [hunk 1]
```diff
@@ -403,19 +390,10 @@ var Vis = View.extend({
...
     if (cartoDBLayerGroup) {
       var endpoint;
-      var mapsApiTemplate;
-      var userName;
-      var statTag;
-      var forceCors = true;
       var configGenerator;
 
       var datasource = data.datasource;
       if (datasource) {
```
Comment (viddo) [inline src/vis/vis.js]: With this change isn't datasource always required? If so, perhaps should throw an error to indicate that it's missing if not set?

**Comment by viddo** [comment]
:+1:

File: src/vis/vis.js (hunk 1 above)
Comment (alonsogarciapablo) [inline src/vis/vis.js]: Yes, good catch!

**Pull request merged**